java/core/src/main/java/org/bondlib/UInt32BondType.java

```diff
@@ -75,7 +75,8 @@ protected final Integer deserializeValue(TaggedDeserializationContext context) t
     }
 
     @Override
-    protected final Integer deserializeValue(UntaggedDeserializationContext context) throws IOException {
+    protected final Integer deserializeValue(
+        UntaggedDeserializationContext context, TypeDef typeDef) throws IOException {
```

Comment:
> TypeDef typeDef [](start = 47, length = 16)

Minor inconsistency: other types have this on a new line.

Reply:
Done.